index.html

```diff
@@ -1,538 +1,571 @@
-          if (deliveryType) {
-            document.querySelector(".metadata #delivery-type-key").innerText = deliveryType === 'residential' ? '🏡' : '🏢';
-            document.querySelector(".metadata #delivery-type-value").innerText = deliveryType.substring(0, 1).toUpperCase() + deliveryType.substring(1);
-          }
-        }
-
-        document.querySelector(".metadata #delivery-address-key").innerText = data.result.address ? '📍' : '';
-        document.querySelector(".metadata #delivery-address-value").innerHTML = data.result.address ? Object.values(data.result.address).filter(line => line !== "").join("<br>") : '';
-
-        document.querySelector(".metadata").classList.remove("invisible");
-      }
-
-      // Hide the "metadata" container
-      function resetMetadata() {
-        document.querySelector(".metadata #confidence-key").innerText = '';
-        document.querySelector(".metadata #confidence-value").innerText = '';
-        document.querySelector(".metadata #delivery-type-key").innerText = '';
-        document.querySelector(".metadata #delivery-type-value").innerText = '';
-
-        document.querySelector(".metadata").classList.add("invisible");
-        document.querySelector("#map").classList.add("hidden");
-      }
-    </script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Experian Address Validation - sample page</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="dist/css/demo.css">
+  <link rel="stylesheet" type="text/css" href="dist/css/experian-address-validation.css">
+  <link rel="stylesheet" type="text/css" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css">
+  <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" defer></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1, maximum-scale=1" />
+  <link rel="shortcut icon" href="./dist/images/favicon.png">
+</head>
+
+<body spellcheck="false">
+  <div class="page-header">
+    <img src="dist/images/logo.png" width="137" height="60" loading="eager" decoding="async" alt="Experian logo">
+  </div>
+
+  <div class="hero-header">
+    <h1>Experian Address Validation</h1>
+  </div>
+
+  <div class="intro">
+    <p>Welcome to our Experian Address Validation demo. Here you can try some of our available search types.</p>
+
+    <form class="token-prompt hidden" onsubmit="addToken(); return false;">
+      Please enter a valid token to get started:<br>
+      <input type="text" name="token" class="token-input"> <button>Go</button><br>
+      <small>Or <a href='https://github.com/experianplc/Experian-Address-Validation#tokens' target="_blank">find out how
+          to get a token</a> if you don't have one.</small>

```

Comment:
Why did you split the message on two rows ?

Reply:
That's the auto format on save

Reply:
not good :)

Reply:
fixed